setup.py

```diff
@@ -61,6 +59,29 @@ def get_extensions():
     return ext_modules
 
 
+def get_model_zoo_configs(exclude: Sequence[str] = []) -> Sequence[Tuple[str, Sequence[str]]]:
```

Comment:
With `package_data=...`, I found that a symlink suffices, and no need to copy. How about just creating one symlink and copy only when this fails (e.g. on windows or some windows file system)?
It will then also include configs for quick_schedules, etc, but that should be fine.

Reply:
What if I `pip install .` and delete the repo later? I haven't tried this but it might fail.

Not sure if symlink gets occupied to `site-packages` or the actual file. Will try and rrport back.

Reply:
When using `package_data` I found that the symlink'ed directory gets copied during installation.